stdlib/runtime/string.gr

```diff
@@ -381,19 +386,23 @@ let rec heapValueToString = (ptr, extraIndents, toplevel) => {
             variantName
           } else {
             let comspace = ", "
-            Memory.incRef(WasmI32.fromGrain(SLEmpty))
```

Comment:
Why doesn't this get an incRef anymore? Looks like this is still needed?

Reply:
oops